### PR TITLE
add whitelisting info to docs

### PR DESCRIPTION
**Pull request by rjnordeen**
@bmchrist

### CODE REVIEW

File: sso.rst [hunk 1]
```diff
@@ -46,3 +53,9 @@ Base DN          The DN we should search in for the user that is logging in
 Filter           Which field should correspond with the login name they use                                         (uid={{login}})
 Firewall?        Is your system behind a firewall that will need us to provide a static IP for whitelisting?        yes/no
 ================ ================================================================================================== =====================================
+
+You may need to whitelist Handshake's IP addresses to access your ldap server please allow the following IPs:
+
+*  ``54.88.136.216``
```
Comment (sgringwe) [inline sso.rst]: @rjnordeen where did you get these IP's from?

Comment (bmchrist) [inline sso.rst]: Also, these will only function for them to whitelist if we instruct our system to use those IPs when accessing their LDAP host

Comment (bmchrist) [inline sso.rst]: he probably got them from https://joinhandshake.atlassian.net/wiki/display/CS/Single+Sign+On

Comment (rjnordeen) [inline sso.rst]: Ya listing them as I'd rather have them do the work first and then us do it after, we'll eventually get to point where they can just check the box of they need whitelisting and that will be setup to use the static IPs, for now let's cut 1 iteration cycle out on our side :)

Comment (rjnordeen) [inline sso.rst]: @sgringwe are those not the right IPs?

Comment (bmchrist) [inline sso.rst]: https://www.dropbox.com/s/m89scowyn0mhi2x/Screenshot%202016-01-04%2013.55.54.png?dl=0

Comment (bmchrist) [inline sso.rst]: They're right

Comment (sgringwe) [inline sso.rst]: @rjnordeen was just making sure that we were on the same page, which we are

**Pull request merged**